common/src/main/java/org/vivecraft/mixin/client_vr/MinecraftVRMixin.java

```diff
@@ -321,6 +321,15 @@ public abstract class MinecraftVRMixin implements MinecraftExtension {
     public void vivecraft$preTickTasks(CallbackInfo ci) {
         if (VRState.vrRunning) {
             ClientDataHolderVR.getInstance().vrPlayer.preTick();
+
+            if (ClientDataHolderVR.getInstance().menuWorldRenderer != null) {
```

Comment:
this should be run when `VRState.vrEnabled` is true, to not be needed to have vr actively running